### PR TITLE
Feature/bar chart

### PR DESCRIPTION
**Pull request by fcollares**
## Description
After the previous vis/judges api route worked well, I went back and added functionality and cleaned up the code. 
Currently the DS team can manually edit what features they want to display on the x-axis, and multiple graphs can be returned at the same time for the web team to parse and implement.
The next step is to decide what we will add to the graph on the DS side and implement those additions.

- Created a visualizations.py file to house new function called get_stacked_bar_chart that creates a bar chart and returns the chart in json format
- Streamlined vis/judges endpoint to use get_stacked_bar_chart to create multiple bar graphs and return all of them in a dictionary in json format
- Added my personal notebook called Labs34_bar_chart

## Type of change

Please delete options that are not relevant.

- [x]  New feature (non-breaking change which adds functionality)
- [ ]  This change requires a documentation update

## Checklist:

- [x]  My code follows the style guidelines of this project
- [x]  I have performed a self-review of my own code
- [x]  I have removed unnecessary comments/console logs from my code
- [x]  I have made corresponding changes to the documentation if necessary (optional)
- [x]  My changes generate no new warnings
- [x]  I have checked my code and corrected any misspellings
- [x]  No duplicate code left within changed files
- [x]  Size of pull request kept to a minimum
- [x]  Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -65,58 +65,30 @@ async def pdf_ocr(uuid: str):
     except ClientError:
         return {"status": f"File not found: {uuid}.pdf"}
 
-from fastapi import Request, FastAPI
```
Comment (KWeatherwalks) [inline app/main.py:68]: Thanks for removing this!  
This import is already handled by line 15.

File: app/main.py [hunk 2]
```diff
@@ -65,58 +65,30 @@ async def pdf_ocr(uuid: str):
...
-
-    # Change the bar mode - stack vs group
-    fig.update_layout(barmode='stack')
-
-    return fig.to_json()
+    chart_1 = get_stacked_bar_chart(df, 'protected_grounds')
+    chart_2 = get_stacked_bar_chart(df, 'gender')
+    chart_3 = get_stacked_bar_chart(df, 'country_of_origin')
+
+    charts_dict = {'bar_protected_grounds': chart_1,
```
Comment (KWeatherwalks) [inline app/main.py:87]: Nice naming of the charts. Need to inform the web team that the JSON will now have the keys `[bar_protected_grounds, bar_gender, bar_country_of_origin]` when they receive this object.

**Review by KWeatherwalks — APPROVED** [review]
This is a much needed extension of the previous work that will deliver more basic visualizations to the web back-end.

File: app/main.py (hunk 2 above)
Comment (fcollares) [reply app/main.py:87]: Good catch. I sent them a DM explaining.

**Review by JRBOH — APPROVED** [review]
We are uploading gender and get_country now. I'm so thankful for the work you and Kevin have done for viz work.

**Pull request merged**